src/stan-users-guide/efficiency-tuning.qmd

```diff
@@ -1346,18 +1375,18 @@ model {
   // priors
   alpha ~ normal(0, 10);
   beta ~ normal(0, 10);
-  sigma ~ cauchy(0, 5);
+  sigma ~ normal(0, 5);
   // likelihood
   for (n in 1:N) {
     y[n] ~ normal(alpha + beta * x[n], sigma);
   }
 }
 ```
 
-
-The data block for the standardized model is identical.  The
-standardized predictors and outputs are defined in the transformed
-data block.
+The data block for the standardized model is identical.
+In the transformed data block the mean and standard deviation
+of the data are computed and saved as global variables,
```

Comment:
I see what you're getting at, but I wouldn't say "global variables" here.  They are transformed data scope, which means they can be used in any block after transformed data.

Reply:
revised.

Reply:
sorry Bob, I mistakenly requested re-review for me and you. I'll re-review now